docker-local/docker-compose.yml

```diff
@@ -336,14 +336,17 @@ services:
       - "dfspa-backend:172.17.0.1"
 
   dfspa-thirdparty-scheme-adapter-inbound:
-    image: "mojaloop/thirdparty-scheme-adapter:v11.15.0"
+    image: "mojaloop/thirdparty-scheme-adapter:v11.18.0"
+    # build: 
```

Comment:
nit: these comments serve any purpose?

Reply:
I would like to keep these comments for a moment to have a shortcut for local building & testing

Reply:
:+1: